apps/admin-gui/src/app/facilities/pages/resource-detail-page/resource-assigned-services/resource-assigned-services.component.html

```diff
@@ -0,0 +1,24 @@
+<h1 class="page-subtitle">{{'RESOURCE_DETAIL.ASSIGNED_SERVICES.TITLE' | translate}}</h1>
+<perun-web-apps-refresh-button (refresh)="loadAllServices()"></perun-web-apps-refresh-button>
+<button mat-flat-button color="accent" (click)="addService()">
+  {{'RESOURCE_DETAIL.ASSIGNED_SERVICES.ADD_SERVICE' | translate}}
+</button>
+<button mat-flat-button color="warn" class="ml-2" [disabled]="selected.selected.length === 0" (click)="removeServices()">
+  {{'RESOURCE_DETAIL.ASSIGNED_SERVICES.REMOVE_SERVICE' | translate}}
+</button>
+<app-debounce-filter
+  [placeholder]="'RESOURCE_DETAIL.ASSIGNED_SERVICES.FILTER_SERVICES'"
+  (filter)="applyFilter($event)">
+</app-debounce-filter>
+<mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
+<div class="mt-3" *ngIf="!loading && assignedServices.length !== 0">
+  <app-services-list
```

Comment:
for some reason the styling of service-list is broken. The card around the table is missing. Try rebase. If that doesnt help change first row of service.list.component.html and change class to:
`class="card mt-3"`